articles/azure-maps/map-add-controls.md

```diff
@@ -110,9 +110,18 @@ Learn more about the classes and methods used in this article:
 > [!div class="nextstepaction"]
 > [PitchControl]
 
+> [!div class="nextstepaction"]
+> [ScaleControl]
+
 > [!div class="nextstepaction"]
 > [StyleControl]
 
+> [!div class="nextstepaction"]
```

Comment:
We don't introduce traffic and traffic legend control in this article, please remove those. Thx.

Reply:
@rbrundritt Please make these edits. Thanks!